tools/pubsys/Cargo.toml

```diff
@@ -19,25 +19,25 @@ log = "0.4"
 parse-datetime = { path = "../../sources/parse-datetime" }
 # Need to bring in reqwest with a TLS feature so tough can support TLS repos.
 reqwest = { version = "0.10.1", default-features = false, features = ["rustls-tls", "blocking"] }
-rusoto_core = "0.45.0"
+rusoto_core = { version = "0.45.0", default-features = false, features = ["rustls"] }
 rusoto_credential = "0.45.0"
-rusoto_ebs = "0.45.0"
-rusoto_ec2 = "0.45.0"
+rusoto_ebs = { version = "0.45.0", default-features = false, features = ["rustls"] }
+rusoto_ec2 = { version = "0.45.0", default-features = false, features = ["rustls"] }
 rusoto_signature = "0.45.0"
-rusoto_ssm = "0.45.0"
-rusoto_sts = "0.45.0"
+rusoto_ssm = { version = "0.45.0", default-features = false, features = ["rustls"] }
+rusoto_sts = { version = "0.45.0", default-features = false, features = ["rustls"] }
 simplelog = "0.8"
-snafu = "0.6"
+snafu = "0.6.9"
```

Comment:
Similarly, is this required?  Do we depend on a snafu bugfix to update tough?

Reply:
Same as above, it was the result of a battle with cargo, not a desire for a specific fix in any given crate.